aten/src/ATen/native/native_functions.yaml

```diff
@@ -4942,6 +4942,15 @@
   dispatch:
     CPU, CUDA: sign_out
 
+- func: signbit(Tensor self, *) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+
+- func: signbit_(Tensor(a!) self) -> Tensor(a!)
+  variants: method
+
+- func: signbit.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
```

Comment:
To fix the xla error, please add a dispatch section here, e.g.
```
dispatch:
CPU: signbit_out
CUDA: signbit_out
```
since the `signbit_out` implementation is done through dispatch stub which won't for xla backend.

Reply:
After the change, the JIT error disappeared and everything seemed to be going well. 😃 
Thank you very much for your guidance. @ailzhang